doc/generating-idyntree-matlab-bindings.md

```diff
@@ -33,7 +33,9 @@ Now, all is left is to compile and generate the bindings.
 
 ## Generating the bindings
 - In the build folder of `iDynTree`, run `ccmake` to open the CMake Gui.
+    - Set the CMake option `SWIG_DIR` to `<swig_install_dir>/share/swig/<version>`.
     - Set the CMake option `SWIG_EXECUTABLE` to `<swig_install_dir>/bin/swig`.
+    - Set the CMake option `SWIG_VERSION` to `<version>`.
```

Comment:
This two steps should not be necessary. They were necessary in your case?

Reply:
yes, otherwise it was keep using the share folder of the swig I have installed in mamba env, resulting in errors when compiling.

Reply:
Ah, good point. Probably it is sufficient if you are doing a clean build, but indeed in general it is better to set it.

Reply:
With the following `CMakeCache.txt`, where I am just setting the `SWIG_EXECUTABLE`  variable, I get the following error when compiling:

```
(robsub) lomoretti@IITICUBLAP226:~/robotology/robotology-superbuild/build/src/iDynTree (master)$ make
[  1%] Built target idyntree-private-fpconv
[ 17%] Built target idyntree-core
[ 36%] Built target idyntree-model
[ 39%] Built target idyntree-modelio-xml
[ 49%] Built target idyntree-modelio
[ 59%] Built target idyntree-estimation
[ 60%] Built target idyntree-solid-shapes
[ 61%] Built target idyntree-high-level
[ 65%] Built target idyntree-inverse-kinematics
[ 85%] Built target idyntree-optimalcontrol
[ 94%] Built target idyntree-visualization
[ 95%] Built target idyntree-model-info
[ 96%] Built target idyntree-model-view
[ 97%] Built target idyntree-model-simplify-shapes
[ 98%] Swig compile ../iDynTree.i for python
/home/lomoretti/mambaforge/envs/robsub/share/swig/4.1.0/typemaps/swigtype.swg:587: Error: Syntax error in input(1).
make[2]: *** [bindings/python/CMakeFiles/iDynTree_swig_compilation.dir/build.make:78: bindings/python/CMakeFiles/iDynTree.dir/iDynTreePYTHON.stamp] Error 1
make[2]: *** Deleting file 'bindings/python/CMakeFiles/iDynTree.dir/iDynTreePYTHON.stamp'
make[1]: *** [CMakeFiles/Makefile2:918: bindings/python/CMakeFiles/iDynTree_swig_compilation.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```


<details open>
    <summary> CMakeCache.txt </summary>

```
(robsub) lomoretti@IITICUBLAP226:~/robotology/robotology-superbuild/build/src/iDynTree (master)$ cat CMakeCache.txt 
# This is the CMakeCache file.
# For build in directory: /home/lomoretti/robotology/robotology-superbuild/build/src/iDynTree
# It was generated by CMake: /home/lomoretti/mambaforge/envs/robsub/bin/cmake
# You can edit this file to change values found and used by cmake.
# If you do not want to change any of the values, simply exit the editor.
# If you do want to change a value, simply edit, save, and exit the editor.
# The syntax for the file is as follows:
# KEY:TYPE=VALUE
# KEY is the name of a variable in the cache.
# TYPE is a hint to GUIs for the type of VALUE, DO NOT EDIT TYPE!.
# VALUE is the current value for the KEY.

########################
# EXTERNAL cache entries
########################

//Build libraries as shared as opposed to static
BUILD_SHARED_LIBS:BOOL=ON

//Path to a program.
CMAKE_ADDR2LINE:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-addr2line

//Path to a program.
CMAKE_AR:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-ar

//Initial cache
CMAKE_BUILD_TYPE:STRING=Debug

//Enable/Disable color output during build.
CMAKE_COLOR_MAKEFILE:BOOL=ON

//CXX compiler
CMAKE_CXX_COMPILER:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-c++

//A wrapper around 'ar' adding the appropriate '--plugin' option
// for the GCC compiler
CMAKE_CXX_COMPILER_AR:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-gcc-ar

//A wrapper around 'ranlib' adding the appropriate '--plugin' option
// for the GCC compiler
CMAKE_CXX_COMPILER_RANLIB:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-gcc-ranlib

//Flags used by the CXX compiler during all build types.
CMAKE_CXX_FLAGS:STRING=-fvisibility-inlines-hidden -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem /home/lomoretti/mambaforge/envs/robsub/include

//Flags used by the CXX compiler during DEBUG builds.
CMAKE_CXX_FLAGS_DEBUG:STRING=-g

//Flags used by the CXX compiler during MINSIZEREL builds.
CMAKE_CXX_FLAGS_MINSIZEREL:STRING=-Os -DNDEBUG

//Flags used by the CXX compiler during RELEASE builds.
CMAKE_CXX_FLAGS_RELEASE:STRING=-O3 -DNDEBUG

//Flags used by the CXX compiler during RELWITHDEBINFO builds.
CMAKE_CXX_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG

//C compiler
CMAKE_C_COMPILER:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-cc

//A wrapper around 'ar' adding the appropriate '--plugin' option
// for the GCC compiler
CMAKE_C_COMPILER_AR:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-gcc-ar

//A wrapper around 'ranlib' adding the appropriate '--plugin' option
// for the GCC compiler
CMAKE_C_COMPILER_RANLIB:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-gcc-ranlib

//Flags used by the C compiler during all build types.
CMAKE_C_FLAGS:STRING=-march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem /home/lomoretti/mambaforge/envs/robsub/include

//Flags used by the C compiler during DEBUG builds.
CMAKE_C_FLAGS_DEBUG:STRING=-g

//Flags used by the C compiler during MINSIZEREL builds.
CMAKE_C_FLAGS_MINSIZEREL:STRING=-Os -DNDEBUG

//Flags used by the C compiler during RELEASE builds.
CMAKE_C_FLAGS_RELEASE:STRING=-O3 -DNDEBUG

//Flags used by the C compiler during RELWITHDEBINFO builds.
CMAKE_C_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_ICUB:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_YARP:BOOL=ON

//Path to a program.
CMAKE_DLLTOOL:FILEPATH=CMAKE_DLLTOOL-NOTFOUND

//Flags used by the linker during all build types.
CMAKE_EXE_LINKER_FLAGS:STRING=-Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections -Wl,--allow-shlib-undefined -Wl,-rpath,/home/lomoretti/mambaforge/envs/robsub/lib -Wl,-rpath-link,/home/lomoretti/mambaforge/envs/robsub/lib -L/home/lomoretti/mambaforge/envs/robsub/lib

//Flags used by the linker during DEBUG builds.
CMAKE_EXE_LINKER_FLAGS_DEBUG:STRING=

//Flags used by the linker during MINSIZEREL builds.
CMAKE_EXE_LINKER_FLAGS_MINSIZEREL:STRING=

//Flags used by the linker during RELEASE builds.
CMAKE_EXE_LINKER_FLAGS_RELEASE:STRING=

//Flags used by the linker during RELWITHDEBINFO builds.
CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO:STRING=

//No help, variable specified on the command line.
CMAKE_EXPORT_COMPILE_COMMANDS:BOOL=TRUE

//Value Computed by CMake.
CMAKE_FIND_PACKAGE_REDIRECTS_DIR:STATIC=/home/lomoretti/robotology/robotology-superbuild/build/src/iDynTree/CMakeFiles/pkgRedirects

//User executables (bin)
CMAKE_INSTALL_BINDIR:PATH=bin

//Read-only architecture-independent data (DATAROOTDIR)
CMAKE_INSTALL_DATADIR:PATH=

//Read-only architecture-independent data root (share)
CMAKE_INSTALL_DATAROOTDIR:PATH=share

//Documentation root (DATAROOTDIR/doc/PROJECT_NAME)
CMAKE_INSTALL_DOCDIR:PATH=

//C header files (include)
CMAKE_INSTALL_INCLUDEDIR:PATH=include

//Info documentation (DATAROOTDIR/info)
CMAKE_INSTALL_INFODIR:PATH=

//Object code libraries (lib)
CMAKE_INSTALL_LIBDIR:PATH=lib

//Program executables (libexec)
CMAKE_INSTALL_LIBEXECDIR:PATH=libexec

//Locale-dependent data (DATAROOTDIR/locale)
CMAKE_INSTALL_LOCALEDIR:PATH=

//Modifiable single-machine data (var)
CMAKE_INSTALL_LOCALSTATEDIR:PATH=var

//Man documentation (DATAROOTDIR/man)
CMAKE_INSTALL_MANDIR:PATH=

//C header files for non-gcc (/usr/include)
CMAKE_INSTALL_OLDINCLUDEDIR:PATH=/usr/include

//Initial cache
CMAKE_INSTALL_PREFIX:PATH=/home/lomoretti/robotology/robotology-superbuild/build/install

//Run-time variable data (LOCALSTATEDIR/run)
CMAKE_INSTALL_RUNSTATEDIR:PATH=

//System admin executables (sbin)
CMAKE_INSTALL_SBINDIR:PATH=sbin

//Modifiable architecture-independent data (com)
CMAKE_INSTALL_SHAREDSTATEDIR:PATH=com

//Read-only single-machine data (etc)
CMAKE_INSTALL_SYSCONFDIR:PATH=etc

//Path to a program.
CMAKE_LINKER:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-ld

//Path to a program.
CMAKE_MAKE_PROGRAM:FILEPATH=/usr/bin/gmake

//Flags used by the linker during the creation of modules during
// all build types.
CMAKE_MODULE_LINKER_FLAGS:STRING=-Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections -Wl,--allow-shlib-undefined -Wl,-rpath,/home/lomoretti/mambaforge/envs/robsub/lib -Wl,-rpath-link,/home/lomoretti/mambaforge/envs/robsub/lib -L/home/lomoretti/mambaforge/envs/robsub/lib

//Flags used by the linker during the creation of modules during
// DEBUG builds.
CMAKE_MODULE_LINKER_FLAGS_DEBUG:STRING=

//Flags used by the linker during the creation of modules during
// MINSIZEREL builds.
CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL:STRING=

//Flags used by the linker during the creation of modules during
// RELEASE builds.
CMAKE_MODULE_LINKER_FLAGS_RELEASE:STRING=

//Flags used by the linker during the creation of modules during
// RELWITHDEBINFO builds.
CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO:STRING=

//Path to a program.
CMAKE_NM:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-nm

//Path to a program.
CMAKE_OBJCOPY:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-objcopy

//Path to a program.
CMAKE_OBJDUMP:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-objdump

//When compiling static libraries generate position independent
// code
CMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON

//No help, variable specified on the command line.
CMAKE_PREFIX_PATH:PATH=/home/lomoretti/robotology/robotology-superbuild/build/install;/home/lomoretti/mambaforge/envs/robsub;/home/lomoretti/mambaforge/envs/robsub/x86_64-conda-linux-gnu/sysroot/usr

//Value Computed by CMake
CMAKE_PROJECT_DESCRIPTION:STATIC=

//Value Computed by CMake
CMAKE_PROJECT_HOMEPAGE_URL:STATIC=

//Value Computed by CMake
CMAKE_PROJECT_NAME:STATIC=iDynTree

//Value Computed by CMake
CMAKE_PROJECT_VERSION:STATIC=11.0.0

//Value Computed by CMake
CMAKE_PROJECT_VERSION_MAJOR:STATIC=11

//Value Computed by CMake
CMAKE_PROJECT_VERSION_MINOR:STATIC=0

//Value Computed by CMake
CMAKE_PROJECT_VERSION_PATCH:STATIC=0

//Value Computed by CMake
CMAKE_PROJECT_VERSION_TWEAK:STATIC=

//Path to a program.
CMAKE_RANLIB:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-ranlib

//Path to a program.
CMAKE_READELF:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-readelf

//Flags used by the linker during the creation of shared libraries
// during all build types.
CMAKE_SHARED_LINKER_FLAGS:STRING=-Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections -Wl,--allow-shlib-undefined -Wl,-rpath,/home/lomoretti/mambaforge/envs/robsub/lib -Wl,-rpath-link,/home/lomoretti/mambaforge/envs/robsub/lib -L/home/lomoretti/mambaforge/envs/robsub/lib

//Flags used by the linker during the creation of shared libraries
// during DEBUG builds.
CMAKE_SHARED_LINKER_FLAGS_DEBUG:STRING=

//Flags used by the linker during the creation of shared libraries
// during MINSIZEREL builds.
CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL:STRING=

//Flags used by the linker during the creation of shared libraries
// during RELEASE builds.
CMAKE_SHARED_LINKER_FLAGS_RELEASE:STRING=

//Flags used by the linker during the creation of shared libraries
// during RELWITHDEBINFO builds.
CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO:STRING=

//Initial cache
CMAKE_SKIP_INSTALL_RPATH:PATH=NO

//Initial cache
CMAKE_SKIP_RPATH:PATH=NO

//Flags used by the linker during the creation of static libraries
// during all build types.
CMAKE_STATIC_LINKER_FLAGS:STRING=

//Flags used by the linker during the creation of static libraries
// during DEBUG builds.
CMAKE_STATIC_LINKER_FLAGS_DEBUG:STRING=

//Flags used by the linker during the creation of static libraries
// during MINSIZEREL builds.
CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL:STRING=

//Flags used by the linker during the creation of static libraries
// during RELEASE builds.
CMAKE_STATIC_LINKER_FLAGS_RELEASE:STRING=

//Flags used by the linker during the creation of static libraries
// during RELWITHDEBINFO builds.
CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO:STRING=

//Path to a program.
CMAKE_STRIP:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/x86_64-conda-linux-gnu-strip

//Path to a program.
CMAKE_TAPI:FILEPATH=CMAKE_TAPI-NOTFOUND

//If this value is on, makefiles will be generated without the
// .SILENT directive, and all commands will be echoed to the console
// during the make.  This is useful for debugging only. With Visual
// Studio IDE projects all commands are done without /nologo.
CMAKE_VERBOSE_MAKEFILE:BOOL=FALSE

//Path to a program.
DOT_EXECUTABLE:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/dot

//Dot tool for use with Doxygen
DOXYGEN_DOT_EXECUTABLE:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/dot

//Doxygen documentation generation tool (https://www.doxygen.nl)
DOXYGEN_EXECUTABLE:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/doxygen

//The directory containing a CMake configuration file for Eigen3.
Eigen3_DIR:PATH=/home/lomoretti/mambaforge/envs/robsub/share/eigen3/cmake

//Compile iDynTree optimal control part.
IDYNTREE_COMPILES_OPTIMALCONTROL:BOOL=ON

//Compile iDynTree tools.
IDYNTREE_COMPILES_TOOLS:BOOL=ON

//No help, variable specified on the command line.
IDYNTREE_COMPILES_YARP_TOOLS:BOOL=OFF

//Compile iDynTree tests
IDYNTREE_COMPILE_TESTS:BOOL=OFF

//Do you want iDynTree to detect and install in the active site-package
// directory? (it could be a system dir)
IDYNTREE_DETECT_ACTIVE_PYTHON_SITEPACKAGES:BOOL=OFF

//Install directory for iDynTree doxygen documentation
IDYNTREE_DOXYGEN_INSTALL_DIR:PATH=/home/lomoretti/robotology/robotology-superbuild/build/src/iDynTree/doc

//Generate a verbose HTML output for Doxygen, including call and
// caller graphs.
IDYNTREE_DOXYGEN_VERBOSE:BOOL=OFF

//Generate doxygen XML output, for use by SWIG
IDYNTREE_DOXYGEN_XML:BOOL=OFF

//Enable RPATH for the library
IDYNTREE_ENABLE_RPATH:BOOL=ON

//Enable if you have the experimental version of SWIG necessary
// for generating the Matlab wrapper
IDYNTREE_GENERATE_MATLAB:BOOL=ON

//Location (relative to the install prefix) in which the Matlab
// mex libraries are installed.
IDYNTREE_INSTALL_MATLAB_LIBDIR:STRING=mex

//Location (relative to the install prefix) in which the Matlab
// .m files are installed.
IDYNTREE_INSTALL_MATLAB_MFILESDIR:STRING=mex

//Location (relative to the install prefix) in which the Octave
// mex libraries are installed.
IDYNTREE_INSTALL_OCTAVE_LIBDIR:STRING=octave

//Location (relative to the install prefix) in which the Octave
// .m files are installed.
IDYNTREE_INSTALL_OCTAVE_MFILESDIR:STRING=octave

//Only produce iDynTree documentation, without compiling
IDYNTREE_ONLY_DOCS:BOOL=OFF

//No help, variable specified on the command line.
IDYNTREE_PYTHON_INSTALL_DIR:UNINITIALIZED=lib/python3.12/site-packages

//Use CMake to install Python pip metadata. Set to false if someone
// else install it.
IDYNTREE_PYTHON_PIP_METADATA_INSTALL:BOOL=ON

//Specify the string to identify the pip Installer. Default: cmake,
// change this if you are using another tool.
IDYNTREE_PYTHON_PIP_METADATA_INSTALLER:STRING=cmake

//Run iDynTree tests with Valgrind
IDYNTREE_RUN_VALGRIND_TESTS:BOOL=OFF

//Build the part of iDynTree that depends on package ALGLIB
IDYNTREE_USES_ALGLIB:BOOL=OFF

//Build the part of iDynTree that depends on package assimp
IDYNTREE_USES_ASSIMP:BOOL=ON

//Build the part of iDynTree that depends on package ICUB
IDYNTREE_USES_ICUB_MAIN:BOOL=OFF

//Compile iDynTree with an internal copy of urdfdom patched to
// avoid Boost dependencies
IDYNTREE_USES_INTERNAL_URDFDOM:BOOL=OFF

//Build the part of iDynTree that depends on package IPOPT
IDYNTREE_USES_IPOPT:BOOL=ON

//Build the part of iDynTree that depends on package Irrlicht
IDYNTREE_USES_IRRLICHT:BOOL=ON

//Do you want to create the Lua bindings
IDYNTREE_USES_LUA:BOOL=OFF

//Do you want to create the MATLAB bindings
IDYNTREE_USES_MATLAB:BOOL=ON

//Build the part of iDynTree that depends on package MeshcatCpp
IDYNTREE_USES_MESHCATCPP:BOOL=OFF

//Do you want to create the OCTAVE bindings
IDYNTREE_USES_OCTAVE:BOOL=OFF

//Build the part of iDynTree that depends on package OsqpEigen
IDYNTREE_USES_OSQPEIGEN:BOOL=ON

//Do you want to create the Python bindings
IDYNTREE_USES_PYTHON:BOOL=ON

//Create iDynTree Python bindings using pybind11
IDYNTREE_USES_PYTHON_PYBIND11:BOOL=OFF

//Build the part of iDynTree that depends on package WORHP
IDYNTREE_USES_WORHP:BOOL=OFF

//Build the part of iDynTree that depends on package YARP
IDYNTREE_USES_YARP:BOOL=OFF

//Additional compiler flags for IPOPT
IPOPT_DEFINITIONS:STRING=

//IPOPT include directory
IPOPT_INCLUDE_DIRS:PATH=/home/lomoretti/mambaforge/envs/robsub/include/coin-or

//IPOPT libraries
IPOPT_LIBRARIES:STRING=

//The directory containing a CMake configuration file for Irrlicht.
Irrlicht_DIR:PATH=Irrlicht_DIR-NOTFOUND

//Path to a file.
Irrlicht_INCLUDE_DIR:PATH=/home/lomoretti/mambaforge/envs/robsub/include/irrlicht

//Path to a library.
Irrlicht_LIBRARY:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/lib/libIrrlicht.so

//Path to a file.
LIBXML2_INCLUDE_DIR:PATH=/home/lomoretti/mambaforge/envs/robsub/include/libxml2

//Path to a library.
LIBXML2_LIBRARY:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/lib/libxml2.so

//Path to a program.
LIBXML2_XMLLINT_EXECUTABLE:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/xmllint

//MatlabDataArray Library
Matlab_DATAARRAY_LIBRARY:FILEPATH=/usr/local/MATLAB/R2023b/bin/glnxa64/libMatlabDataArray.so

//MatlabEngine Library
Matlab_ENGINE_LIBRARY:FILEPATH=/usr/local/MATLAB/R2023b/extern/bin/glnxa64/libMatlabEngine.so

//Path to a file.
Matlab_INCLUDE_DIRS:PATH=/usr/local/MATLAB/R2023b/extern/include

//Matlab MEX extension provider
Matlab_MEXEXTENSIONS_PROG:FILEPATH=/usr/local/MATLAB/R2023b/bin/mexext

//Extensions for the mex targets (automatically given by Matlab)
Matlab_MEX_EXTENSION:STRING=mexa64

//Path to a library.
Matlab_MEX_LIBRARY:FILEPATH=/usr/local/MATLAB/R2023b/bin/glnxa64/libmex.so

//Path to a library.
Matlab_MX_LIBRARY:FILEPATH=/usr/local/MATLAB/R2023b/bin/glnxa64/libmx.so

//Matlab installation root path
Matlab_ROOT_DIR:PATH=/usr/local/MATLAB/R2023b

//Path to a file.
OPENGL_EGL_INCLUDE_DIR:PATH=OPENGL_EGL_INCLUDE_DIR-NOTFOUND

//Path to a file.
OPENGL_GLES2_INCLUDE_DIR:PATH=OPENGL_GLES2_INCLUDE_DIR-NOTFOUND

//Path to a file.
OPENGL_GLES3_INCLUDE_DIR:PATH=OPENGL_GLES3_INCLUDE_DIR-NOTFOUND

//Path to a file.
OPENGL_GLX_INCLUDE_DIR:PATH=/home/lomoretti/mambaforge/envs/robsub/x86_64-conda-linux-gnu/sysroot/usr/include

//Path to a file.
OPENGL_INCLUDE_DIR:PATH=/home/lomoretti/mambaforge/envs/robsub/x86_64-conda-linux-gnu/sysroot/usr/include

//Path to a library.
OPENGL_egl_LIBRARY:FILEPATH=OPENGL_egl_LIBRARY-NOTFOUND

//Path to a library.
OPENGL_gl_LIBRARY:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/x86_64-conda-linux-gnu/sysroot/usr/lib/libGL.so

//Path to a library.
OPENGL_gles2_LIBRARY:FILEPATH=OPENGL_gles2_LIBRARY-NOTFOUND

//Path to a library.
OPENGL_gles3_LIBRARY:FILEPATH=OPENGL_gles3_LIBRARY-NOTFOUND

//Path to a library.
OPENGL_glu_LIBRARY:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/lib/libGLU.so

//Path to a library.
OPENGL_glx_LIBRARY:FILEPATH=OPENGL_glx_LIBRARY-NOTFOUND

//Path to a library.
OPENGL_opengl_LIBRARY:FILEPATH=OPENGL_opengl_LIBRARY-NOTFOUND

//Path to a file.
OPENGL_xmesa_INCLUDE_DIR:PATH=OPENGL_xmesa_INCLUDE_DIR-NOTFOUND

//The directory containing a CMake configuration file for OsqpEigen.
OsqpEigen_DIR:PATH=/home/lomoretti/robotology/robotology-superbuild/build/install/lib/cmake/OsqpEigen

//Arguments to supply to pkg-config
PKG_CONFIG_ARGN:STRING=

//pkg-config executable
PKG_CONFIG_EXECUTABLE:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/bin/pkg-config

//Path to a file.
SWIG_DIR:PATH=/home/lomoretti/mambaforge/envs/robsub/share/swig/4.1.0

//Path to a program.
SWIG_EXECUTABLE:FILEPATH=/home/lomoretti/bindings/bin/swig

//Swig version
SWIG_VERSION:STRING=4.1.0

//The directory containing a CMake configuration file for assimp.
assimp_DIR:PATH=/home/lomoretti/mambaforge/envs/robsub/lib/cmake/assimp-5.3

//The directory containing a CMake configuration file for glfw3.
glfw3_DIR:PATH=/home/lomoretti/mambaforge/envs/robsub/lib/cmake/glfw3

//Value Computed by CMake
iDynTreeBindings_BINARY_DIR:STATIC=/home/lomoretti/robotology/robotology-superbuild/build/src/iDynTree/bindings

//Value Computed by CMake
iDynTreeBindings_IS_TOP_LEVEL:STATIC=OFF

//Value Computed by CMake
iDynTreeBindings_SOURCE_DIR:STATIC=/home/lomoretti/robotology/robotology-superbuild/src/iDynTree/bindings

//Value Computed by CMake
iDynTree_BINARY_DIR:STATIC=/home/lomoretti/robotology/robotology-superbuild/build/src/iDynTree

//Value Computed by CMake
iDynTree_IS_TOP_LEVEL:STATIC=ON

//Value Computed by CMake
iDynTree_SOURCE_DIR:STATIC=/home/lomoretti/robotology/robotology-superbuild/src/iDynTree

//Path to a library.
ipopt_PATH:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/lib/libipopt.so

//The directory containing a CMake configuration file for osqp.
osqp_DIR:PATH=/home/lomoretti/robotology/robotology-superbuild/build/install/lib/cmake/osqp

//Path to a library.
pkgcfg_lib_PC_LIBXML_xml2:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/lib/libxml2.so

//Path to a library.
pkgcfg_lib__PC_IPOPT_ipopt:FILEPATH=/home/lomoretti/mambaforge/envs/robsub/lib/libipopt.so


########################
# INTERNAL cache entries
########################

//ADVANCED property for variable: CMAKE_ADDR2LINE
CMAKE_ADDR2LINE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_AR
CMAKE_AR-ADVANCED:INTERNAL=1
//STRINGS property for variable: CMAKE_BUILD_TYPE
CMAKE_BUILD_TYPE-STRINGS:INTERNAL=Debug;Release;MinSizeRel;RelWithDebInfo
//This is the directory where this CMakeCache.txt was created
CMAKE_CACHEFILE_DIR:INTERNAL=/home/lomoretti/robotology/robotology-superbuild/build/src/iDynTree
//Major version of cmake used to create the current loaded cache
CMAKE_CACHE_MAJOR_VERSION:INTERNAL=3
//Minor version of cmake used to create the current loaded cache
CMAKE_CACHE_MINOR_VERSION:INTERNAL=28
//Patch version of cmake used to create the current loaded cache
CMAKE_CACHE_PATCH_VERSION:INTERNAL=3
//ADVANCED property for variable: CMAKE_COLOR_MAKEFILE
CMAKE_COLOR_MAKEFILE-ADVANCED:INTERNAL=1
//Path to CMake executable.
CMAKE_COMMAND:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/bin/cmake
//Path to cpack program executable.
CMAKE_CPACK_COMMAND:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/bin/cpack
//Path to ctest program executable.
CMAKE_CTEST_COMMAND:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/bin/ctest
//ADVANCED property for variable: CMAKE_CXX_COMPILER
CMAKE_CXX_COMPILER-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_COMPILER_AR
CMAKE_CXX_COMPILER_AR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_COMPILER_RANLIB
CMAKE_CXX_COMPILER_RANLIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS
CMAKE_CXX_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS_DEBUG
CMAKE_CXX_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS_MINSIZEREL
CMAKE_CXX_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS_RELEASE
CMAKE_CXX_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS_RELWITHDEBINFO
CMAKE_CXX_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_COMPILER
CMAKE_C_COMPILER-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_COMPILER_AR
CMAKE_C_COMPILER_AR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_COMPILER_RANLIB
CMAKE_C_COMPILER_RANLIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS
CMAKE_C_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_DEBUG
CMAKE_C_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_MINSIZEREL
CMAKE_C_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_RELEASE
CMAKE_C_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_RELWITHDEBINFO
CMAKE_C_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_DLLTOOL
CMAKE_DLLTOOL-ADVANCED:INTERNAL=1
//Path to cache edit program executable.
CMAKE_EDIT_COMMAND:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/bin/ccmake
//Executable file format
CMAKE_EXECUTABLE_FORMAT:INTERNAL=ELF
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS
CMAKE_EXE_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_DEBUG
CMAKE_EXE_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_MINSIZEREL
CMAKE_EXE_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_RELEASE
CMAKE_EXE_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//Name of external makefile project generator.
CMAKE_EXTRA_GENERATOR:INTERNAL=
//Name of generator.
CMAKE_GENERATOR:INTERNAL=Unix Makefiles
//Generator instance identifier.
CMAKE_GENERATOR_INSTANCE:INTERNAL=
//Name of generator platform.
CMAKE_GENERATOR_PLATFORM:INTERNAL=
//Name of generator toolset.
CMAKE_GENERATOR_TOOLSET:INTERNAL=
//Test CMAKE_HAVE_LIBC_PTHREAD
CMAKE_HAVE_LIBC_PTHREAD:INTERNAL=
//Have library pthreads
CMAKE_HAVE_PTHREADS_CREATE:INTERNAL=
//Have library pthread
CMAKE_HAVE_PTHREAD_CREATE:INTERNAL=1
//Source directory with the top level CMakeLists.txt file for this
// project
CMAKE_HOME_DIRECTORY:INTERNAL=/home/lomoretti/robotology/robotology-superbuild/src/iDynTree
//ADVANCED property for variable: CMAKE_INSTALL_BINDIR
CMAKE_INSTALL_BINDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_DATADIR
CMAKE_INSTALL_DATADIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_DATAROOTDIR
CMAKE_INSTALL_DATAROOTDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_DOCDIR
CMAKE_INSTALL_DOCDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_INCLUDEDIR
CMAKE_INSTALL_INCLUDEDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_INFODIR
CMAKE_INSTALL_INFODIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_LIBDIR
CMAKE_INSTALL_LIBDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_LIBEXECDIR
CMAKE_INSTALL_LIBEXECDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_LOCALEDIR
CMAKE_INSTALL_LOCALEDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_LOCALSTATEDIR
CMAKE_INSTALL_LOCALSTATEDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_MANDIR
CMAKE_INSTALL_MANDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_OLDINCLUDEDIR
CMAKE_INSTALL_OLDINCLUDEDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_RUNSTATEDIR
CMAKE_INSTALL_RUNSTATEDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_SBINDIR
CMAKE_INSTALL_SBINDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_SHAREDSTATEDIR
CMAKE_INSTALL_SHAREDSTATEDIR-ADVANCED:INTERNAL=1
//Install .so files without execute permission.
CMAKE_INSTALL_SO_NO_EXE:INTERNAL=1
//ADVANCED property for variable: CMAKE_INSTALL_SYSCONFDIR
CMAKE_INSTALL_SYSCONFDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_LINKER
CMAKE_LINKER-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MAKE_PROGRAM
CMAKE_MAKE_PROGRAM-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS
CMAKE_MODULE_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_DEBUG
CMAKE_MODULE_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL
CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_RELEASE
CMAKE_MODULE_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_NM
CMAKE_NM-ADVANCED:INTERNAL=1
//number of local generators
CMAKE_NUMBER_OF_MAKEFILES:INTERNAL=22
//ADVANCED property for variable: CMAKE_OBJCOPY
CMAKE_OBJCOPY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_OBJDUMP
CMAKE_OBJDUMP-ADVANCED:INTERNAL=1
//Platform information initialized
CMAKE_PLATFORM_INFO_INITIALIZED:INTERNAL=1
//ADVANCED property for variable: CMAKE_RANLIB
CMAKE_RANLIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_READELF
CMAKE_READELF-ADVANCED:INTERNAL=1
//Path to CMake installation.
CMAKE_ROOT:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/share/cmake-3.28
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS
CMAKE_SHARED_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_DEBUG
CMAKE_SHARED_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL
CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_RELEASE
CMAKE_SHARED_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SKIP_INSTALL_RPATH
CMAKE_SKIP_INSTALL_RPATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SKIP_RPATH
CMAKE_SKIP_RPATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS
CMAKE_STATIC_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_DEBUG
CMAKE_STATIC_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL
CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_RELEASE
CMAKE_STATIC_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STRIP
CMAKE_STRIP-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_TAPI
CMAKE_TAPI-ADVANCED:INTERNAL=1
//uname command
CMAKE_UNAME:INTERNAL=/usr/bin/uname
//ADVANCED property for variable: CMAKE_VERBOSE_MAKEFILE
CMAKE_VERBOSE_MAKEFILE-ADVANCED:INTERNAL=1
//Compiler support for a deprecated attribute
COMPILER_HAS_DEPRECATED:INTERNAL=1
//Test COMPILER_HAS_DEPRECATED_ATTR
COMPILER_HAS_DEPRECATED_ATTR:INTERNAL=1
//Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY
COMPILER_HAS_HIDDEN_INLINE_VISIBILITY:INTERNAL=1
//Test COMPILER_HAS_HIDDEN_VISIBILITY
COMPILER_HAS_HIDDEN_VISIBILITY:INTERNAL=1
//Test CXX_HAS_WNO_DEPRECATED
CXX_HAS_WNO_DEPRECATED:INTERNAL=1
//Test CXX_HAS_WNO_DEPRECATED_DECLARATIONS
CXX_HAS_WNO_DEPRECATED_DECLARATIONS:INTERNAL=1
//ADVANCED property for variable: DOXYGEN_DOT_EXECUTABLE
DOXYGEN_DOT_EXECUTABLE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: DOXYGEN_EXECUTABLE
DOXYGEN_EXECUTABLE-ADVANCED:INTERNAL=1
//Details about finding Doxygen
FIND_PACKAGE_MESSAGE_DETAILS_Doxygen:INTERNAL=[/home/lomoretti/mambaforge/envs/robsub/bin/doxygen][cfound components: doxygen dot ][v1.10.0()]
//Details about finding IPOPT
FIND_PACKAGE_MESSAGE_DETAILS_IPOPT:INTERNAL=[/home/lomoretti/mambaforge/envs/robsub/lib/libipopt.so][v()]
//Details about finding Irrlicht
FIND_PACKAGE_MESSAGE_DETAILS_Irrlicht:INTERNAL=[/home/lomoretti/mambaforge/envs/robsub/include/irrlicht][/home/lomoretti/mambaforge/envs/robsub/lib/libIrrlicht.so][v()]
//Details about finding LibXml2
FIND_PACKAGE_MESSAGE_DETAILS_LibXml2:INTERNAL=[/home/lomoretti/mambaforge/envs/robsub/lib/libxml2.so][/home/lomoretti/mambaforge/envs/robsub/include/libxml2][v2.12.5()]
//Details about finding Matlab
FIND_PACKAGE_MESSAGE_DETAILS_Matlab:INTERNAL=[/usr/local/MATLAB/R2023b/extern/include][/usr/local/MATLAB/R2023b/bin/glnxa64/libmex.so][mexa64][/usr/local/MATLAB/R2023b][/usr/local/MATLAB/R2023b/bin/glnxa64/libmx.so][c ][v23.2.0.2485118()]
//Details about finding OpenGL
FIND_PACKAGE_MESSAGE_DETAILS_OpenGL:INTERNAL=[/home/lomoretti/mambaforge/envs/robsub/x86_64-conda-linux-gnu/sysroot/usr/lib/libGL.so][/home/lomoretti/mambaforge/envs/robsub/x86_64-conda-linux-gnu/sysroot/usr/include][c ][v()]
//Details about finding Python3
FIND_PACKAGE_MESSAGE_DETAILS_Python3:INTERNAL=[/home/lomoretti/mambaforge/envs/robsub/bin/python3.12][/home/lomoretti/mambaforge/envs/robsub/include/python3.12][/home/lomoretti/mambaforge/envs/robsub/lib/python3.12/site-packages/numpy/core/include][cfound components: Interpreter Development.Module NumPy ][v3.12.2()]
//Details about finding SWIG
FIND_PACKAGE_MESSAGE_DETAILS_SWIG:INTERNAL=[/home/lomoretti/bindings/bin/swig][/home/lomoretti/mambaforge/envs/robsub/share/swig/4.1.0][c ][v4.1.0()]
//Details about finding Threads
FIND_PACKAGE_MESSAGE_DETAILS_Threads:INTERNAL=[TRUE][v()]
//Test HAS_MINUS_PTHREAD
HAS_MINUS_PTHREAD:INTERNAL=1
//ADVANCED property for variable: IDYNTREE_DOXYGEN_INSTALL_DIR
IDYNTREE_DOXYGEN_INSTALL_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: IDYNTREE_DOXYGEN_VERBOSE
IDYNTREE_DOXYGEN_VERBOSE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: IDYNTREE_DOXYGEN_XML
IDYNTREE_DOXYGEN_XML-ADVANCED:INTERNAL=1
//ADVANCED property for variable: IDYNTREE_ENABLE_RPATH
IDYNTREE_ENABLE_RPATH-ADVANCED:INTERNAL=1
//MODIFIED property for variable: IDYNTREE_GENERATE_MATLAB
IDYNTREE_GENERATE_MATLAB-MODIFIED:INTERNAL=ON
//ADVANCED property for variable: IDYNTREE_INSTALL_MATLAB_LIBDIR
IDYNTREE_INSTALL_MATLAB_LIBDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: IDYNTREE_INSTALL_MATLAB_MFILESDIR
IDYNTREE_INSTALL_MATLAB_MFILESDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: IDYNTREE_INSTALL_OCTAVE_LIBDIR
IDYNTREE_INSTALL_OCTAVE_LIBDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: IDYNTREE_INSTALL_OCTAVE_MFILESDIR
IDYNTREE_INSTALL_OCTAVE_MFILESDIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: IDYNTREE_ONLY_DOCS
IDYNTREE_ONLY_DOCS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: IDYNTREE_PYTHON_PIP_METADATA_INSTALL
IDYNTREE_PYTHON_PIP_METADATA_INSTALL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: IDYNTREE_RUN_VALGRIND_TESTS
IDYNTREE_RUN_VALGRIND_TESTS-ADVANCED:INTERNAL=1
//MODIFIED property for variable: IDYNTREE_USES_MATLAB
IDYNTREE_USES_MATLAB-MODIFIED:INTERNAL=ON
//ADVANCED property for variable: IPOPT_DEFINITIONS
IPOPT_DEFINITIONS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: IPOPT_INCLUDE_DIRS
IPOPT_INCLUDE_DIRS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: IPOPT_LIBRARIES
IPOPT_LIBRARIES-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Irrlicht_INCLUDE_DIR
Irrlicht_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Irrlicht_LIBRARY
Irrlicht_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: LIBXML2_INCLUDE_DIR
LIBXML2_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: LIBXML2_LIBRARY
LIBXML2_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: LIBXML2_XMLLINT_EXECUTABLE
LIBXML2_XMLLINT_EXECUTABLE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Matlab_DATAARRAY_LIBRARY
Matlab_DATAARRAY_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Matlab_ENGINE_LIBRARY
Matlab_ENGINE_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Matlab_INCLUDE_DIRS
Matlab_INCLUDE_DIRS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Matlab_MEXEXTENSIONS_PROG
Matlab_MEXEXTENSIONS_PROG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Matlab_MEX_EXTENSION
Matlab_MEX_EXTENSION-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Matlab_MEX_LIBRARY
Matlab_MEX_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: Matlab_MX_LIBRARY
Matlab_MX_LIBRARY-ADVANCED:INTERNAL=1
//internal matlab location for the discovered version
Matlab_PROG_VERSION_STRING_AUTO_DETECT:INTERNAL=/usr/local/MATLAB/R2023b/bin/matlab
//last Matlab root dir location
Matlab_ROOT_DIR_LAST_CACHED:INTERNAL=/usr/local/MATLAB/R2023b
//Matlab version (automatically determined)
Matlab_VERSION_STRING_INTERNAL:INTERNAL=23.2.0.2485118
//ADVANCED property for variable: OPENGL_EGL_INCLUDE_DIR
OPENGL_EGL_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: OPENGL_GLES2_INCLUDE_DIR
OPENGL_GLES2_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: OPENGL_GLES3_INCLUDE_DIR
OPENGL_GLES3_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: OPENGL_GLX_INCLUDE_DIR
OPENGL_GLX_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: OPENGL_INCLUDE_DIR
OPENGL_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: OPENGL_egl_LIBRARY
OPENGL_egl_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: OPENGL_gl_LIBRARY
OPENGL_gl_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: OPENGL_gles2_LIBRARY
OPENGL_gles2_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: OPENGL_gles3_LIBRARY
OPENGL_gles3_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: OPENGL_glu_LIBRARY
OPENGL_glu_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: OPENGL_glx_LIBRARY
OPENGL_glx_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: OPENGL_opengl_LIBRARY
OPENGL_opengl_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: OPENGL_xmesa_INCLUDE_DIR
OPENGL_xmesa_INCLUDE_DIR-ADVANCED:INTERNAL=1
PC_LIBXML_CFLAGS:INTERNAL=-I/home/lomoretti/mambaforge/envs/robsub/include/libxml2;-I/home/lomoretti/mambaforge/envs/robsub/include
PC_LIBXML_CFLAGS_I:INTERNAL=
PC_LIBXML_CFLAGS_OTHER:INTERNAL=
PC_LIBXML_FOUND:INTERNAL=1
PC_LIBXML_INCLUDEDIR:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/include
PC_LIBXML_INCLUDE_DIRS:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/include/libxml2;/home/lomoretti/mambaforge/envs/robsub/include
PC_LIBXML_LDFLAGS:INTERNAL=-L/home/lomoretti/mambaforge/envs/robsub/lib;-lxml2
PC_LIBXML_LDFLAGS_OTHER:INTERNAL=
PC_LIBXML_LIBDIR:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/lib
PC_LIBXML_LIBRARIES:INTERNAL=xml2
PC_LIBXML_LIBRARY_DIRS:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/lib
PC_LIBXML_LIBS:INTERNAL=
PC_LIBXML_LIBS_L:INTERNAL=
PC_LIBXML_LIBS_OTHER:INTERNAL=
PC_LIBXML_LIBS_PATHS:INTERNAL=
PC_LIBXML_MODULE_NAME:INTERNAL=libxml-2.0
PC_LIBXML_PREFIX:INTERNAL=/home/lomoretti/mambaforge/envs/robsub
PC_LIBXML_STATIC_CFLAGS:INTERNAL=-I/home/lomoretti/mambaforge/envs/robsub/include/libxml2;-I/home/lomoretti/mambaforge/envs/robsub/include
PC_LIBXML_STATIC_CFLAGS_I:INTERNAL=
PC_LIBXML_STATIC_CFLAGS_OTHER:INTERNAL=
PC_LIBXML_STATIC_INCLUDE_DIRS:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/include/libxml2;/home/lomoretti/mambaforge/envs/robsub/include
PC_LIBXML_STATIC_LDFLAGS:INTERNAL=-L/home/lomoretti/mambaforge/envs/robsub/lib;-lxml2;-lz;-llzma;-lm;-licui18n;-licuuc;-licudata;-lpthread;-ldl;-lm
PC_LIBXML_STATIC_LDFLAGS_OTHER:INTERNAL=
PC_LIBXML_STATIC_LIBDIR:INTERNAL=
PC_LIBXML_STATIC_LIBRARIES:INTERNAL=xml2;z;lzma;m;icui18n;icuuc;icudata;pthread;dl;m
PC_LIBXML_STATIC_LIBRARY_DIRS:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/lib
PC_LIBXML_STATIC_LIBS:INTERNAL=
PC_LIBXML_STATIC_LIBS_L:INTERNAL=
PC_LIBXML_STATIC_LIBS_OTHER:INTERNAL=
PC_LIBXML_STATIC_LIBS_PATHS:INTERNAL=
PC_LIBXML_VERSION:INTERNAL=2.12.5
PC_LIBXML_libxml-2.0_INCLUDEDIR:INTERNAL=
PC_LIBXML_libxml-2.0_LIBDIR:INTERNAL=
PC_LIBXML_libxml-2.0_PREFIX:INTERNAL=
PC_LIBXML_libxml-2.0_VERSION:INTERNAL=
//ADVANCED property for variable: PKG_CONFIG_ARGN
PKG_CONFIG_ARGN-ADVANCED:INTERNAL=1
//ADVANCED property for variable: PKG_CONFIG_EXECUTABLE
PKG_CONFIG_EXECUTABLE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: SWIG_DIR
SWIG_DIR-ADVANCED:INTERNAL=1
//MODIFIED property for variable: SWIG_DIR
SWIG_DIR-MODIFIED:INTERNAL=ON
//ADVANCED property for variable: SWIG_EXECUTABLE
SWIG_EXECUTABLE-ADVANCED:INTERNAL=1
//MODIFIED property for variable: SWIG_EXECUTABLE
SWIG_EXECUTABLE-MODIFIED:INTERNAL=ON
//ADVANCED property for variable: SWIG_VERSION
SWIG_VERSION-ADVANCED:INTERNAL=1
//MODIFIED property for variable: SWIG_VERSION
SWIG_VERSION-MODIFIED:INTERNAL=ON
//linker supports push/pop state
_CMAKE_LINKER_PUSHPOP_STATE_SUPPORTED:INTERNAL=TRUE
//CMAKE_INSTALL_PREFIX during last run
_GNUInstallDirs_LAST_CMAKE_INSTALL_PREFIX:INTERNAL=/home/lomoretti/robotology/robotology-superbuild/build/install
_PC_IPOPT_CFLAGS:INTERNAL=-I/home/lomoretti/mambaforge/envs/robsub/include/coin-or
_PC_IPOPT_CFLAGS_I:INTERNAL=
_PC_IPOPT_CFLAGS_OTHER:INTERNAL=
_PC_IPOPT_FOUND:INTERNAL=1
_PC_IPOPT_INCLUDEDIR:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/include/coin-or
_PC_IPOPT_INCLUDE_DIRS:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/include/coin-or
_PC_IPOPT_LDFLAGS:INTERNAL=-L/home/lomoretti/mambaforge/envs/robsub/lib;-lipopt
_PC_IPOPT_LDFLAGS_OTHER:INTERNAL=
_PC_IPOPT_LIBDIR:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/lib
_PC_IPOPT_LIBRARIES:INTERNAL=ipopt
_PC_IPOPT_LIBRARY_DIRS:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/lib
_PC_IPOPT_LIBS:INTERNAL=
_PC_IPOPT_LIBS_L:INTERNAL=
_PC_IPOPT_LIBS_OTHER:INTERNAL=
_PC_IPOPT_LIBS_PATHS:INTERNAL=
_PC_IPOPT_MODULE_NAME:INTERNAL=ipopt
_PC_IPOPT_PREFIX:INTERNAL=/home/lomoretti/mambaforge/envs/robsub
_PC_IPOPT_STATIC_CFLAGS:INTERNAL=-I/home/lomoretti/mambaforge/envs/robsub/include/coin-or
_PC_IPOPT_STATIC_CFLAGS_I:INTERNAL=
_PC_IPOPT_STATIC_CFLAGS_OTHER:INTERNAL=
_PC_IPOPT_STATIC_INCLUDE_DIRS:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/include/coin-or
_PC_IPOPT_STATIC_LDFLAGS:INTERNAL=-L/home/lomoretti/mambaforge/envs/robsub/lib;-lipopt
_PC_IPOPT_STATIC_LDFLAGS_OTHER:INTERNAL=
_PC_IPOPT_STATIC_LIBDIR:INTERNAL=
_PC_IPOPT_STATIC_LIBRARIES:INTERNAL=ipopt
_PC_IPOPT_STATIC_LIBRARY_DIRS:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/lib
_PC_IPOPT_STATIC_LIBS:INTERNAL=
_PC_IPOPT_STATIC_LIBS_L:INTERNAL=
_PC_IPOPT_STATIC_LIBS_OTHER:INTERNAL=
_PC_IPOPT_STATIC_LIBS_PATHS:INTERNAL=
_PC_IPOPT_VERSION:INTERNAL=3.14.14
_PC_IPOPT_ipopt_INCLUDEDIR:INTERNAL=
_PC_IPOPT_ipopt_LIBDIR:INTERNAL=
_PC_IPOPT_ipopt_PREFIX:INTERNAL=
_PC_IPOPT_ipopt_VERSION:INTERNAL=
//Compiler reason failure
_Python3_Compiler_REASON_FAILURE:INTERNAL=
_Python3_DEVELOPMENT_MODULE_SIGNATURE:INTERNAL=b87754c99bf4c7af56ac1bddea22b683
//Development reason failure
_Python3_Development_REASON_FAILURE:INTERNAL=
//Path to a program.
_Python3_EXECUTABLE:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/bin/python3.12
//Path to a file.
_Python3_INCLUDE_DIR:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/include/python3.12
//Python3 Properties
_Python3_INTERPRETER_PROPERTIES:INTERNAL=Python;3;12;1;64;;cpython-312-x86_64-linux-gnu;abi3;/home/lomoretti/mambaforge/envs/robsub/lib/python3.12;/home/lomoretti/mambaforge/envs/robsub/lib/python3.12;/home/lomoretti/mambaforge/envs/robsub/lib/python3.12/site-packages;/home/lomoretti/mambaforge/envs/robsub/lib/python3.12/site-packages
_Python3_INTERPRETER_SIGNATURE:INTERNAL=b3c39cea63dc7ce132d6fa94bae4b9cc
//Interpreter reason failure
_Python3_Interpreter_REASON_FAILURE:INTERNAL=
_Python3_NUMPY_SIGNATURE:INTERNAL=e3c5ee7dc466798c6ae84eb91f5a94cf
//Path to a file.
_Python3_NumPy_INCLUDE_DIR:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/lib/python3.12/site-packages/numpy/core/include
__pkg_config_arguments_PC_LIBXML:INTERNAL=QUIET;libxml-2.0
__pkg_config_arguments__PC_IPOPT:INTERNAL=QUIET;ipopt
__pkg_config_checked_PC_LIBXML:INTERNAL=1
__pkg_config_checked__PC_IPOPT:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PC_LIBXML_xml2
pkgcfg_lib_PC_LIBXML_xml2-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib__PC_IPOPT_ipopt
pkgcfg_lib__PC_IPOPT_ipopt-ADVANCED:INTERNAL=1
prefix_result:INTERNAL=/home/lomoretti/mambaforge/envs/robsub/lib
```

</details>